app/src/main/kotlin/eu/quiqua/gloomhaven/statstracker/features/stats/viewModel/StatsViewModel.kt

```diff
@@ -17,7 +20,42 @@ class StatsViewModel @Inject constructor(application: android.app.Application) :
     }
 
     @Inject
-    lateinit var model: String
+    lateinit var stats: Stats
 
-    fun getStats(): String = model
+    val mutableHp: MutableLiveData<Int> = MutableLiveData()
+    val mutableXp: MutableLiveData<Int> = MutableLiveData()
+
+    fun currentHp(): LiveData<Int> {
```

Comment:
this method is never used

Reply:
it is! 
https://github.com/quiqua/gloomhaven-stats-tracker/blob/680ab7c5003b7a6953d6aabc89540dfc81c0434b/app/src/main/res/layout/activity_stats.xml#L34
 :)

Reply:
and then does it need to be a method to be accessed from there? otherwise I would just change to a val with getter :)